src/x509_parser.rs

```diff
@@ -247,7 +247,8 @@ fn parse_extensions(i:&[u8]) -> IResult<&[u8],Vec<X509Extension>,BerError> {
 }
 
 
-fn parse_tbs_certificate(i:&[u8]) -> IResult<&[u8],TbsCertificate,BerError> {
+fn parse_tbs_certificate(i:&[u8]) -> IResult<&[u8],(TbsCertificate,&[u8]),BerError> {
+    const BER_HEADER_SIZE: usize = 4;
```

Comment:
I'm a bit uncomfortable hard-coding the header size: even if it is canonical (DER), length could be greater (or smaller ?) for big (tiny) certificates. There should be a way to build the size.